ruby-challenges.rb

```diff
@@ -3,33 +3,99 @@
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter.
 
-beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+# beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
-letter_o = 'o'
+# letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
-letter_t = 't'
+# letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+# def return_word array, string
+#     array.select do |value|
+#         value.include?(string)
+#         end
+#     end
+#     p return_word(beverages_array, letter_o)
+#     p return_word(beverages_array, letter_t) 
+
+
+
+
 
 # -------------------2) Create a method that takes in an array of numbers and returns the sum of the numbers. Use the test variables provided.
 
-nums_array1 = [42, 7, 27]
-# Expected output: 76
+# nums_array1 = [42, 7, 27]
+# # Expected output: 76
+
+# nums_array2 = [25, 17, 47, 11]
+# # Expected output: 100
 
-nums_array2 = [25, 17, 47, 11]
-# Expected output: 100
+# def sum_of_numbers array
+#     return array.sum()
+# end
+# p sum_of_numbers(nums_array1)
+# p sum_of_numbers(nums_array2)
 
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
-# Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
+
+# class Bike
+#     attr_accessor :model, :wheels, :current_speed
+#     def initialize(model)
+#         @model=model
+#         @wheels = 2
+#         @current_speed = 0
+#     end
+#     def bike_info 
+#         "The #{@model} bike has #{@wheels} wheels and is going #{@current_speed} mph." 
+#     end
+# end
+# bike_type=Bike.new('Mongoose')
+# p bike.bike_info
+# "The Mongoose bike has 2 wheels and is going 0 mph."
 
 
+# Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
+
+# -----------------------------------------------------------------------------
 
 # -------------------3b) Add the ability to pedal faster and brake. The pedal_faster method should increase the speed by a given amount. The brake method should decrease the speed by a given amount. The bike cannot go negative speeds.
 
+class Bike
+    attr_accessor :model, :wheels, :current_speed, :brake
+    def initialize(current_speed, brake)
+        @model= 'Mongoose'
+        @wheels = 2
+        @current_speed = 0
+        @brake = 0
+    end
```

Comment:
You won't want to hardcode Mongoose as a model, this should be a value that is passed in when you create a new instance of the class.  Ex: my_bike = Bike.new('Mongoose').  Also, brake should not be it's own instance variable, it is a method that will update current_speed.  I would recommend reviewing the [Ruby Classes](https://github.com/learn-academy-2022-delta/Syllabus/blob/main/ruby/classes-objects.md) section in the syllabus.